{
    "data_folder_path": "data/CVE-2014-3429",
    "pull_requests_urls": [
        "https://github.com/ipython/ipython/pull/4845"
    ],
    "cve_name": "CVE-2014-3429",
    "repo": "ipython/ipython",
    "warnings": {
        "cve": "CVE-2014-3429",
        "warnings": []
    },
    "files_extension_stats": {
        ".py": 1
    },
    "file_number": 1,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 37,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}